CHANGELOG.md

```diff
@@ -16,6 +16,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The `authenticate` function now supports two-factor authentication.
 - Support `byName()` for the `gateId` argument in `updateGate()`.
 - Introduce `lintr` as a linter with an Actions workflow
+- Add internal functions for gating (mirroring `gata`)
+  - getEllipsePoints
+  - fitEllipsePoints
+  - covarToParameters
```

Comment:
Minor: Don't need to mention internal functions in the changelog.

Reply:
:+1:

Reply:
Still pending.